packages/suite/src/middlewares/suite/messageSystemMiddleware.ts

```diff
@@ -43,10 +44,19 @@ const messageSystemMiddleware =
                     enabledNetworks,
                 },
             });
-
             const categorizedValidMessages = categorizeMessages(validMessages);
 
+            const validExperiments = getValidExperiments(config, {
```

Comment:
It's a nitpick and too late, I guess, so feel free to ignore, but since the validation params should be consistent for getValidMessages few lines above and here, it would be nice to deduplicate it (put it in some constant and pass that for example). And the mapping to ids (.map(item => item.id)) can happen inside getValidExperiments.

Reply:
Can be resolved after freeze right? So I will merge it and we can change it later

Reply:
There was no reason to merge it before freeze 🙉 But of course we can do is as a followup...